ui/src/main/webapp/components/Navbar.js

```diff
@@ -43,7 +43,7 @@ const HelpDialog = props => {
   return (
     <Dialog open={open} onClose={handleClose} maxWidth='md'>
       <DialogTitle variant='h6' id='help-dialog-title'>
-        {'Welcome to the Project Charleston BETA'}
```

Comment:
This removes it from the welcome dialog but not the navbar at the top of the page. Looks like there is one more removal needed

Reply:
Whoops, removed.